zinnia/admin/widgets.py

```diff
@@ -79,8 +79,7 @@ def get_tags(self):
         """
         Returns the list of tags to auto-complete.
         """
-        return [tag.name for tag in
-                Tag.objects.usage_for_model(Entry)]
+        return [tag.name for tag in Tag.objects.all()]
```

Comment:
Hi, are you sure about this ? Why dropping feature to limit tag listing to Zinnia entries only ?